dataRetriever/resolvers/headerResolver.go

```diff
@@ -88,6 +96,18 @@ func (hdrRes *HeaderResolver) SetEpochHandler(epochHandler dataRetriever.EpochHa
 	return nil
 }
 
+func (hdrRes *HeaderResolver) topicNameForAntiflood() string {
+	prefix := factory.ShardBlocksTopic
+	if hdrRes.shardCoordinator.SelfId() > hdrRes.shardCoordinator.NumberOfShards() {
```

Comment:
Please add a TODO here as this won't work on metachain nodes and intra shard header resolvers.
This should have been done on the header interceptor rather than header resolver.

Reply:
done